### ISSUE TITLE
Make upload size configurable via ENV Vars

### ISSUE THREAD

**Issue by 1letter** [plone/plone-backend#150]
If i want to allow upload files greater 1GB i must patch `zope.ini` and `zope.conf` inside the container. that is not a handy solution.

i patch the zope.ini in my container image 

```
[server:main]
# set to 3GB upload size
max_request_body_size = 3221225472
```

and i patch the zope.conf with:

```
<dos_protection>
    form-memory-limit 3GB
    form-disk-limit 3GB
    form-memfile-limit 4KB
</dos_protection>
```

**Comment by 1letter** [comment]
@fredvd @ericof i can't create a branch nor a PR here in the repro. should i fork this repro?

**Comment by 1letter** [comment]
@ericof mean the "thumb emoji" do a fork ?

**Comment by fredvd** [comment]
1letter What is your proposed fix to make this a configurable parameter in plone-backend?

* The restrictions  of this repo are on purpose more limited/strict than other repositories under plone because this repo generates the example containers, so ai-team/release team / release managers do the modifications here. 

*  I'm all for improving the plone-backend/plone-frontend container images, but these container images are still examples and not meant to be the ultimate image setup for any possible use case.  Using this image as a FROM source in your own Dockerfile and customise it there for individual projects IS the suggested way and nothing is wrong with that.  

At the same time:  if these values can be parameterised easily using environment variables, thats a nice win and improvement.

**Comment by 1letter** [comment]
@fredvd see my patch example above. This can be parameterised per default in the backend image. 3 new parameters and some minor substitutions in the skeleteon templates.

**Comment by erral** [comment]
@1letter I also think that we need this. Please add a PR and I will be happy to review it.

For instance when creating a new project using cookieplone-templates, we can set the required configuration in a `instance.yaml` file that then will be used to produce `zope.conf` and `zope.ini`, how can we do something similar with this image?

**Comment by stevepiercy** [comment]
The only people who can write to this repo are members of the Installers Team and GitHub organization Admins. Are these permissions deliberate?

@erral and @fredvd, @1letter lacks permission to write to this repo, so he will need to create a fork and push from there.

**Comment by fredvd** [comment]
Yes, these permissions are deliberate, as this repo has workflows that create official images.  Please create a PR from a fork

**Comment by fredvd** [comment]
@1letter Is this still something you want to  contribute to with a PR?

**Comment by davisagli** [comment]
@fredvd The zope.conf part of this was done in https://github.com/plone/plone-backend/pull/158

The zope.ini part needs more investigation: https://github.com/plone/plone-backend/pull/158#issuecomment-2623356664
